docs/contributor/prdoc.md

```diff
@@ -18,24 +18,33 @@ A `.prdoc` file is a YAML file with a defined structure (ie JSON Schema). Please
 1. Open a Pull Request and get the PR number.
 1. Generate the file with `prdoc generate <PR_NUMBER>`. The output filename will be printed.
 1. Optional: Install the `prdoc/schema_user.json` schema in your editor, for example
-[VsCode](https://github.com/paritytech/prdoc?tab=readme-ov-file#schemas).
+   [VsCode](https://github.com/paritytech/prdoc?tab=readme-ov-file#schemas).
 1. Edit your `.prdoc` file according to the [Audience](#pick-an-audience) and [SemVer](#record-semver-changes) sections.
 1. Check your prdoc with `prdoc check -n <PR_NUMBER>`. This is optional since the CI will also check it.
 
 > **Tip:** GitHub CLI and jq can be used to provide the number of your PR to generate the correct file:  
 > `prdoc generate $(gh pr view --json number | jq '.number') -o prdoc`
 
+Alternatively you can call the prdoc from PR via `/cmd prdoc` (see args with `/cmd prdoc --help`) in a comment to PR to trigger it from CI.
+
+Options:
+
+- `pr`: The PR number to generate the PrDoc for.
+- `audience`: The audience of whom the changes may concern.
+- `bump`: A default bump level for all crates. The PrDoc will likely need to be edited to reflect the actual changes after generation.
+- `overwrite`: Whether to overwrite any existing PrDoc.
```

Comment:
```suggestion
- `force`: Whether to overwrite any existing PrDoc.
```

Although `overwrite` does make more sense 😅 I will probably do another iteration after this one - also to make the `--pr` argument optional, since it can be inferred from the current context of the comment.

Reply:
good catch, i just copied it from your original PR :)

Reply:
[77dad4f](https://github.com/paritytech/polkadot-sdk/pull/5661/commits/77dad4fb46959d0578306e532b54dda7c3840e12)